string actionName
string[] argVals
---
int64 success_bool
